### ISSUE TITLE
Trailing newline detection doesn't seem to work

### ISSUE THREAD

**Issue by kylef** [realm/SwiftLint#153]
swiftlint is telling me the file should have a single trailing new line, it already does.

```shell
$ swiftlint lint
/Users/kyle/Projects/cocode/palaver-osx/PalaverTests/PalaverTests.swift:36: warning: Trailing Newline Violation: File should have a single trailing newline

$ tail -n4 /Users/kyle/Projects/cocode/palaver-osx/PalaverTests/PalaverTests.swift
        }
    }
}

$
```

**Comment by keith** [comment]
In this example you have 2 trailing newline characters:

```
}\n
\n
```

**Comment by kylef** [comment]
Yes, but only *one* blank trailing new-line.

I was expecting this to behave like `Style/TrailingBlankLines` does in rubocop.

**Comment by keith** [comment]
Ah yes, this is ensuring that there is a single *newline* character at the end of the file.
